libcob/fbdb.c

```diff
@@ -766,24 +779,31 @@ ix_bdb_write_internal (cob_file *f, const int rewrite, const int opt)
 	int			close_cursor, ret;
 
 	close_cursor = bdb_open_cursor (f, 1);
+	ret = COB_STATUS_00_SUCCESS;
 
 	/* Check duplicate alternate keys */
 	if (!rewrite) {
 		if (f->nkeys > 1 && check_alt_keys (f, 0)) {
+#if 0	/* CHECKME: RW-Branch 1324 dropped that check */
+			if (close_cursor) {
+				bdb_close_cursor (f);
+			}
```

Comment:
this CHECK (twice here) should be added to the list of things to be inspected after the merge (at the very end of that list ;-)

Reply:
Done.